aws-cloudfront-domain-redirect/module_test.go

```diff
@@ -0,0 +1,18 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/chanzuckerberg/go-misc/tftest"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+)
+
+func TestModule(t *testing.T) {
+	t.Skip("remove this for real tests")
+	test := tftest.Test{
+		Setup:    func(t *testing.T) *terraform.Options { return nil },
```

Comment:
I would like to see a test here. I know that you aren't set up to run tests in the cztack accounts, so I will take it on.